content/en/docs/concepts/configuration/secret.md

```diff
@@ -89,15 +104,21 @@ kind: Secret
 metadata:
   annotations:
     kubectl.kubernetes.io/last-applied-configuration: { ... }
-  creationTimestamp: 2016-01-22T18:41:56Z
+  creationTimestamp: 2020-01-22T18:41:56Z
   name: mysecret
   namespace: default
   resourceVersion: "164619"
   uid: cfee02d6-c137-11e5-8d73-42010af00002
 type: Opaque
 ```
 
-## Using Secrets
+This example showed a Secret with two keys in the `data` field: `username` and `password`.
```

Comment:
This example shows a ...

Reply:
Maybe “The above example shows&hellip;"?

Reply:
I think it is best to leave out words that indicate where to find the content on the page (above, below).

Reply:
I want to use “showed” because I'm worried that readers will otherwise read on, looking for that sentence to be _followed_ by a manifest.